extensions/amp-mustache/0.2/amp-mustache.js

```diff
@@ -53,13 +53,19 @@ export class AmpMustache extends AMP.BaseTemplate {
     /** @private @const {!JsonObject} */
     this.nestedTemplates_ = dict();
 
-    const content = templateContentClone(this.element);
-    this.processNestedTemplates_(content);
-    const container = this.element.ownerDocument.createElement('div');
-    container.appendChild(content);
+    /** @private {string} */
+    this.template_ = '';
+    if (this.element.tagName == 'TEMPLATE') {
+      const content = templateContentClone(this.element);
+      this.processNestedTemplates_(content);
+      const container = this.element.ownerDocument.createElement('div');
+      container.appendChild(content);
+      this.template_ = container./*OK*/innerHTML;
+    } else if (this.element.tagName == 'SCRIPT') {
+      //TODO(alabiaga): handle nested templates.
```

Comment:
This is lower priority, remove and file a tracking issue. No need to address yet. :)

Reply:
done filed #19450